couler/argo.py

```diff
@@ -242,10 +242,37 @@ def create_s3_artifact(
     )
 
 
+def get_existing_secret(
```

Comment:
get_secret should be fine

Reply:
I changed it to obtain_secret. The tests wouldn't let it pass otherwise. Because of the following [line](https://github.com/nooraangelva/couler/blob/master/couler/argo.py#L35)